### PR TITLE
feat: graceful shutdown server

### PR DESCRIPTION
**Pull request by islishude**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import { ApiPromise } from '@polkadot/api';
 import { WsProvider } from '@polkadot/rpc-provider';
 import { json } from 'express';
+import * as shutdown from 'http-graceful-shutdown';
```
Comment (dvdplm) [inline src/main.ts:21]:
```suggestion
import gracefulShutdown from 'http-graceful-shutdown';
```

**Review by dvdplm — CHANGES_REQUESTED** [review]
The description is lacking; I'm not sure I understand what problem is solved by this PR? Can you please provide a motivation for why you need this?

File: src/main.ts (hunk 1 above)
Comment (islishude) [reply src/main.ts:21]: the package has no default export,so here can't use default import

Comment (islishude) [reply src/main.ts:21]: the compiled result

屏幕快照 2020-10-15 下午6 14 16

**Comment by islishude** [comment]
close,there's not much need for this

**Pull request closed without merging**